### ISSUE TITLE
Commenting needs improvements on accessibility

### ISSUE THREAD

**Issue by juhakujala** [City-of-Helsinki/kerrokantasi-ui#715]
There's some issues currently on comments section when using a screen reader:
- Liking a comment doesn't work with a screen reader.
   - There's currently no indication about the functionality of the like button since there's no text in the like button.
- When opening any sub-comments the comment hierarchy is not clear for a screen reader user.
   - It is hard to know when a comment thread ends and a new one starts

Other improvement ideas:
- Comments cannot be browsed in the "heading level", you need to go through every element.
- Text strings that indicate leaving a comment and opening sub-comments could be made more understandable
- Currently the like button is the first element on a single comment. This should be moved after the comment so that the ordering of elements makes more sense. You should first read the comment and after that decide if you like it.
- Could the date be more simplified so that for screen readers (and other users) it would tell e.g. "John Doe, 2 hours ago", instead of the full date and time being announced. It also reads the mid bullet that is currently between the name and the date.
